examples/benchmarks/run_sort_by_keys_performance.py

```
@@ -113,14 +116,106 @@ def run_performance_sweep():
+    current_run = 0
+
+    for _, N in enumerate(particle_counts):
+        current_run += 1
+
+        print(
+            f"[{current_run:2d}/{total_runs}] Running N={N:8d}...",
+            end=" ",
+        )
+
+        start_time = time.time()
+        min_time, max_time, mean_time = benchmark_u32_pow2_len(N)
```

Ruff (0.16.2)

[warning] Unpacked variable `max_time` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)